test/Makefile.defs

```diff
@@ -157,7 +157,7 @@ ifeq ($(CBL),1)
 endif
 
 OMP_FLAGS += $(EXTRA_OMP_FLAGS)
-OMP_FLAGS += -D__OFFLOAD_ARCH_$(INSTALLED_GPU)__
+OMP_FLAGS += -D__OFFLOAD_ARCH_$(AOMP_GPU)__
```

Comment:
I added a PR for test/Makefile.defs that supports AOMP_GPU as well as ROCR_VISIBLE_DEVICES. See https://github.com/ROCm-Developer-Tools/aomp/pull/457. I would prefer that instead of the switch from INSTALLED_GPU -> AOMP_GPU.